features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/ResumeSessionOOMScenario.m

```diff
@@ -4,14 +4,20 @@ @implementation ResumeSessionOOMScenario
 
 - (void)startBugsnag {
     self.config.autoTrackSessions = NO;
+    BugsnagErrorTypes *errorTypes = [BugsnagErrorTypes new];
+    self.config.enabledErrorTypes = errorTypes;
     [super startBugsnag];
 }
 
 - (void)run {
     [Bugsnag startSession];
-    [Bugsnag notify:[NSException exceptionWithName:@"foo" reason:nil userInfo:nil]];
-    [Bugsnag pauseSession];
-    [Bugsnag resumeSession];
+    // This test has determinism issues with ordering of payloads and batching of event payloads
```

Comment:
I'd be very keen to put a more robust solution in that waiting a set period of time.  Almost certainly something for a separate task, but would be good to put some thought to what it would look like.